src/examples/dune

```diff
@@ -136,18 +76,31 @@
  (action
   (run ocaml-protoc --pp --binary --ml_out ./ %{deps})))
 
+(test
+ (name orgchart_ml)
+ (modules orgchart orgchart_ml)
+ (package ocaml-protoc)
+ (libraries pbrt))
+
+(include dune.inc)
+
 (rule
  (alias runtest)
  (action
-  (diff orgchart.ml.expected orgchart.ml)))
+  (diff dune.inc dune.inc.gen)))
```

Comment:
is there a reason we also diff this?

Reply:
Yes, the diff aids in updating generated rules. It allows for easy promotion of new rules to `dune.inc` via `dune promote`, executable directly from the editor.